content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst

```diff
@@ -0,0 +1,189 @@
+window appears, from which a backorder can be created. Click :guilabel:`Create Backorder` to split
+the manufacturing order into two separate orders, with the reference tags *WH/MO/XXXXX-001* and
+*WH/MO/XXXXX-002*.
+
+.. image:: manufacturing_backorders/create-backorder-button.png
+   :align: center
+   :alt: The Create Backorder button on the "You produced less than initial demand" pop-up window.
+
+Order *001* contains the items that have been manufactured, and is closed immediately. Order *002*
+is the backorder that contains the items that have yet to be manufactured, and remains open, to be
+completed at a later date.
+
+Once the remaining units can be manufactured, navigate to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, and then select the backorder manufacturing order. If all of
+the remaining units are manufactured immediately, simply click :guilabel:`Validate` to close the
+order.
+
+If only some of the remaining units are manufactured immediately, create another backorder for the
+remainder by following the steps detailed in this section.
+
+Create a backorder from tablet view
+===================================
+
+Backorders for manufacturing orders can also be created from the work order tablet view.
+
+.. note::
+   In order to use tablet view, the *Work Orders* setting must be enabled. To enable it, navigate to
+   :menuselection:`Manufacturing --> Configuration --> Settings`. On the :guilabel:`Settings` page,
+   enable the checkbox next to :guilabel:`Work Orders`, then click :guilabel:`Save` to save the
+   change. This makes the :guilabel:`Work Orders` tab appear on manufacturing orders, from which the
+   tablet view can be opened.
+
+   .. image:: manufacturing_backorders/work-orders-setting.png
+      :align: center
+      :alt: The Work Orders setting on the Manufacturing settings page.
+
+To create a backorder from the tablet view, begin by navigating to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`. Select a manufacturing order with a quantity of two or more or
+create one by clicking :guilabel:`Create`.
+
+If a new manufacturing order is created, select a product from the :guilabel:`Product` drop-down
+menu and enter a quantity of two or more in the :guilabel:`Quantity` field, then click
+:guilabel:`Confirm` to confirm the order.
+
+After confirming the manufacturing order, select the :guilabel:`Work Orders` tab and click the
+:guilabel:`📱 (tablet view)` button located on the line of the first work order to enter the tablet view.
+
+.. image:: manufacturing_backorders/tablet-view-button.png
+   :align: center
+   :alt: The tablet view button for a work order on a manufacturing order.
+
+Once in tablet view, enter the quantity being manufactured immediately in the :guilabel:`Units`
+field at the top left of the tablet view.
+
+.. image:: manufacturing_backorders/units-field.png
```

Comment:
Checking: shouldn't the initial value in **Units** count be less than the total for the work order (otherwise how is a backorder created it 5/5 units are produced?)